Раздел 1. Задачи 1,2,3,4,5

Vitalyanova opened the pull request.

Vitalyanova committed:
Раздел 1. Задача 1. Факториал

jsru commented:
Добавляю преподавателя (@koalex) для код-ревью.

Vitalyanova committed:
Задача 2. Валидация имени.
Merge branch 'master' of https://github.com/js-tasks-ru/jsbasic-20220124_vita1980

Vitalyanova committed:
Раздел 1. Задача 2. Валидация имени

jsru commented:
Решение было обновлено, посмотрим что скажет @koalex

Vitalyanova committed:
Добавление Раздел 1. Задача 3. Первый символ заглавный.
Merge branch 'master' of https://github.com/js-tasks-ru/jsbasic-20220124_vita1980

Vitalyanova committed:
Раздел 1. Задача 3. Первый символ - заглавный.

Vitalyanova committed:
Добавление Раздел 1. Задача 4. Проверка на спам
Merge branch 'master' of https://github.com/js-tasks-ru/jsbasic-20220124_vita1980

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

Vitalyanova committed:
Раздел 1. Задача 4. Проверка на спам

jsru commented:
Решение было обновлено, посмотрим что скажет @koalex

Vitalyanova committed:
Добавление Раздел 1. Задача 5. Усечение строки
Merge branch 'master' of https://github.com/js-tasks-ru/jsbasic-20220124_vita1980

Vitalyanova committed:
Раздел 1. Задача 5. Усечение строки

jsru commented:
Решение было обновлено, посмотрим что скажет @koalex
